ios/sdk/src/JitsiMeet.m

```diff
@@ -119,6 +123,14 @@ - (BOOL)application:(UIApplication *)app
 
 #pragma mark - Utility methods
 
+- (void)instantiateReactNativeBridge {
+    _bridgeWrapper = [[RCTBridgeWrapper alloc] init];
```

Comment:
maybe I should add a check to skip if already instantiated

Reply:
Yeah let's do that please.